installer/dmg/scripts/postinstall

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env bash
+
+# Variables the installer application automatically passes along
+# $0 : Returns the path to the script
+# $1 : Returns the path to the package
+# $2 : Returns the target location (for example: /Applications)
+# $3 : Returns the target volume (for example: /Volumes/Macintosh HD)
+
+#############
+### Setup ###
+#############
+TARGET_VOLUME=$3
+LOG_FILE="/tmp/postinstall_AtlasSQL_ODBC.log"
+
+# Log an error
+err(){
+    echo "Err: $@" | tee -a $LOG_FILE
+}
+
+# Log an general info
+info(){
+    if [[ ! -e $LOG_FILE ]]; then
+        touch $LOG_FILE
+    fi
+     echo "Info: $@" | tee -a $LOG_FILE
+}
+
+# Setup the log file
+if [[ ! -e $LOG_FILE ]]; then
+    # Create a new log file
+    touch $LOG_FILE
+else
+    # Clean up the existing file
+    > $LOG_FILE 
+fi
+
+ODBC_PATH="/Library/ODBC"
+if [[ ! -z $TARGET_VOLUME ]]; then
+    ODBC_PATH="$TARGET_VOLUME/Library/ODBC"
+fi
+INI="$ODBC_PATH/odbcinst.ini"
+info "Drivers configuration will be added to $INI"
+INSTALL_ROOT="/Library/MongoDB/AtlasSQL ODBC/"
+# Get the latest installed version
+LATEST_VERSION=$(ls "$INSTALL_ROOT/"| grep -E "[0-9]\.[0-9]\.[0-9]" | sort | cut -d '/' -f2 | tail -n 1)
```

Comment:
umh, so for Windows, we consider on major and minor version and patch updates override the installed driver if any. Both for Mac, we keep all versions.
Shouldn't we have the same approach for Mac, to make things easier?

Reply:
> Ensuring we have consistent names is going to be very important. Can we dynamically calculate the name and version from Cargo and sub that in to text files/vars?

We set the name once. It’s never changing. I think we just need to get it the same.

Reply:
I'm not sure why that reply to @terakilobyte ended up in this thread. Github is wonky.

Reply:
@nbagnard we can switch to using major.minor. I cannot recall why we didn't do that in the BIC.